packages/server/modules/emails/services/verification/request.ts

```diff
@@ -43,7 +43,9 @@ async function createNewVerification(
   if (user.verified)
     throw new EmailVerificationRequestError("User's email is already verified")
 
-  const verificationId = await deleteOldAndInsertNewVerification(user.email)
+  const verificationId = await deleteOldAndInsertNewVerificationFactory({ db })(
```

Comment:
I think this function should be injected, it should be instantiated in resolvers right?

Reply:
@fabis94 I think there was a misunderstanding.
I meant changing the function everywhere it is used so we don't need to change it again in the same work.
So, it is changing through all its usage paths.
But if you already made the work like that I'm fine merging it like this